python/python/bystro/beanstalkd/messages.py

```diff
@@ -13,7 +13,8 @@
 
 # seconds; default AWS load balancer TTL is 60 seconds
 # set the interval 2 seconds beforehand to make room for timing / network delays
-QUEUE_HEARTBEAT_INTERVAL = max(1, float(os.getenv("BYSTRO_BEANSTALKD_HEARTBEAT_INTERVAL", "20")) - 2)
+QUEUE_HEARTBEAT_INTERVAL = max(1, float(os.getenv("BYSTRO_BEANSTALKD_HEARTBEAT_INTERVAL", "30")))
+QUEUE_PRODUCER_TIMEOUT = max(1, float(os.getenv("BYSTRO_BEANSTALKD_PRODUCER_TIMEOUT", "10")))
 
 
 class Event(str, Enum):
```

Comment:
One note on the python client. In testing, the python beanstalkd client does client-side timeouts if the operation doesn't succeed within the connection (socket) timeout time (here `QUEUE_PRODUCER_TIMEOUT`). So unlike python and javascript, we don't need to enforce our own client-side timeout.